src/apps/components/AppDetailsSettingsPage/useAppConfigLoader.ts

```diff
@@ -0,0 +1,75 @@
+import { AppFragment } from "@saleor/fragments/types/AppFragment";
+import { MutableRefObject, useEffect, useRef } from "react";
+import urlJoin from "url-join";
+
+export type UseAppConfigLoaderCallbacks = Record<
+  "onLoad" | "onError",
+  () => void
+>;
+
+function fixRelativeScriptSrc(origin: string) {
+  return (node: HTMLScriptElement) => {
+    // Using node.getAttribute beacuse node.src returns absolute path
+    const src = node.getAttribute("src");
+    if (src?.startsWith("/")) {
+      node.src = urlJoin(origin, src);
+    }
+  };
+}
+
+async function _fetchAndSetContent(
```

Comment:
I don't think we need to prefix this with `_` as we're not exporting it anywhere